auto_show/models.py

```diff
@@ -1,20 +1,20 @@
 from django.db import models
 from django.urls import reverse
 
-from additional.models import BaseData, MainData
+from abstract.abstract_models import BaseData, MainData
 
 
 class AutoShow(BaseData, MainData):
-    """Model for AutoShow"""
+    wish_car = models.TextField(
```

Comment:
If u want to store some structered data it's better to JSONField
But if I understood correctly, you should create Foreign key on wish car